backend/server/src/question.rs

```diff
@@ -0,0 +1,81 @@
+use crate::database::{
+    models::{OrganisationUser, Question, QuestionResponse, Role, UpdateQuestionInput, User},
+    Database,
+};
+
+use rocket::{
+    delete,
+    form::Form,
+    get, put,
+    serde::{json::Json, Serialize},
+};
+
+use std::convert::From;
+
+#[derive(Serialize)]
+pub enum QuestionError {
+    QuestionNotFound,
+    UpdateFailed,
+    InsufficientPermissions,
+}
+
+#[get("/<question_id>")]
+pub async fn get_question(
+    db: Database,
```

Comment:
does this work without taking _user: User as a param? It definitely could but just want to make sure its been tested